airbyte-integrations/connectors/source-faker/source_faker/spec.json

```
@@ -43,6 +43,30 @@
         "minimum": 1,
         "default": 4,
         "order": 4
+      },
+      "wide_data_set_columns": {
+        "title": "Wide Data Set Columns",
+        "description": "How many columns should be in a simulated wide data set? i.e. A value of 10 will produce a 10 column table.",
+        "type": "integer",
+        "minimum": 2,
+        "default": 14,
+        "order": 5
+      },
+      "wide_data_set_tables": {
+        "title": "Wide Data Set Tables",
+        "description": "How many copies of the Wide Dataset column should be created?",
```

```suggestion
        "description": "How many copies of the Wide Dataset stream should be created?",
```

(assuming I understood the `source.py` thing correctly)

---

:doh: thank you